### PR TITLE
Resolve test failures

### PR DESCRIPTION
**Pull request by bwohlberg**
Resolve bm3d/bm4d denoiser test failures.

### CODE REVIEW

File: scico/denoiser.py [hunk 1]
```diff
@@ -60,9 +67,9 @@ def bm3d(x: JaxArray, sigma: float, is_rgb: bool = False):
         raise RuntimeError("Package bm3d is required for use of this function.")
 
     if is_rgb is True:
-        bm3d_eval = tunibm3d.bm3d_rgb
+        bm3d_eval = lambda x, sigma: tbm3d.bm3d_rgb(x, sigma, profile=profile)
     else:
-        bm3d_eval = tunibm3d.bm3d
+        bm3d_eval = lambda x, sigma: tbm3d.bm3d(x, sigma, profile=profile)
```
Comment (Michael-T-McCann) [inline scico/denoiser.py:72]: see next comment

File: scico/denoiser.py [hunk 2]
```diff
@@ -8,23 +8,29 @@
...
 except ImportError:
     have_bm3d = False
+    BM3DProfile = Any
 else:
     have_bm3d = True
+    from bm3d.profiles import BM3DProfile  # type: ignore
 
 try:
-    import bm4d as tunibm4d
+    import bm4d as tbm4d
```
[the remark this answers is not in the document]
Comment (bwohlberg) [reply scico/denoiser.py:27]: Tampere University - the home of the `bm3d` and `bm4d` packages. Maybe `tubm3d` etc. would be better.

File: scico/denoiser.py [hunk 3]
```diff
@@ -116,14 +123,15 @@ def bm4d(x: JaxArray, sigma: float):
...
+        profile: Parameter configuration for BM4D.
 
     Returns:
         Denoised output.
     """
     if not have_bm4d:
         raise RuntimeError("Package bm4d is required for use of this function.")
 
-    bm4d_eval = tunibm4d.bm4d
+    bm4d_eval = lambda x, sigma: tbm4d.bm4d(x, sigma, profile=profile)
```
[the remark this answers is not in the document]
Comment (bwohlberg) [reply scico/denoiser.py:134]: I hadn't noticed. PEP8 has an issue with lambda?

[the remark this answers is not in the document]
Comment (Michael-T-McCann) [reply scico/denoiser.py]: "Always use a def statement instead of an assignment statement that binds a lambda expression directly to an identifier"

**Review by Michael-T-McCann — APPROVED** [review]

**Pull request merged**